### PR TITLE
Update React to 15.6.2

### PR DESCRIPTION
**Pull request by dLobatog**
* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

After https://github.com/theforeman/foreman/pull/4877 was merged, this should be updated here too.

### CODE REVIEW

**Comment by ohadlevy** [comment]
should this pr also delete unused packages ? or do they get removed some how?

File: comps/comps-foreman-rhel7.xml [hunk 1]
```diff
@@ -394,7 +394,6 @@
       <packagereq type="default">tfm-rubygem-trollop-doc</packagereq>
       <packagereq type="default">tfm-rubygem-unf-doc</packagereq>
       <packagereq type="default">tfm-rubygem-unf_ext-doc</packagereq>
-      <packagereq type="default">tfm-rubygem-unicode-doc</packagereq>
```
Comment (mmoll) [inline comps/comps-foreman-rhel7.xml]: looks unrelated?

Comment (dLobatog) [reply comps/comps-foreman-rhel7.xml]: Yes, it was created automatically when running `comps_doc.sh` - the package doesn't exist so it may as well be removed from comps

**Comment by dLobatog** [comment]
Updated to use npm_cache (hardcoded name) instead of /tmp/npm_cache_%{name}-%{version}-%{release} , which seems to cause issues.
